extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -374,6 +395,23 @@ export class ProjectsController {
 		telemetryMeasures.buildDuration = buildEndTime - buildStartTime;
 		telemetryProps.buildSucceeded = (dacpacPath !== '').toString();
 
+		let databaseSource: string | undefined;
+		let databaseSourceValues = project.getRawDatabaseSourceValues();
+
+		for (let databaseSourceIndex = 0; databaseSourceIndex < databaseSourceValues.length; databaseSourceIndex++) {
```

Comment:
Given that this is duplicated code I'd suggest making a helper function that does this logic (including the suggestions I made above as well)